modules/ocl/cl_geo_map_handler.h

```diff
@@ -55,6 +55,7 @@ class GeoKernelParamCallback
     virtual PointFloat2 get_left_scale_factor () = 0;
     virtual PointFloat2 get_right_scale_factor () = 0;
 
+    virtual const BowlDataConfig &get_bowl_config () = 0;
```

Comment:
It doesn't make sense to add get_bowl_config.
GeoMapHandler should not be aware of BowlConfig. if have to add a interface. maybe try 
virtual float get_stable_y_ratio();
The ratio must have a default value for other generic cases.

this ratio can be set in 360_stitching or fisheye.